ecl/hql/hqlexpr.cpp

```diff
@@ -15590,7 +15596,7 @@ IHqlExpression * convertToSimpleAggregate(IHqlExpression * expr)
     loop
     {
         node_operator op = ds->getOperator();
-        if ((op != no_sorted) && (op != no_distributed) && (op != no_preservemeta) && (op != no_alias_scope))
+        if ((op != no_sorted) && (op != no_distributed) && (op != no_unordered) && (op != no_preservemeta) && (op != no_alias_scope))
```

Comment:
Haven't really looked to see what this code is doing but should it perhaps be (op == no_unordered)? My logic is derived from !no_sorted being present and that semantically, sorted = ordered = !unordered => !sorted = unordered.